packages/jbrowse-plugin-apollo/src/components/ManageUsers.tsx

```diff
@@ -0,0 +1,115 @@
+import { AbstractSessionModel, AppRootModel } from '@jbrowse/core/util'
+import DeleteIcon from '@mui/icons-material/Delete'
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+} from '@mui/material'
+import {
+  DataGrid,
+  GridActionsCellItem,
+  GridColumns,
+  GridRowId,
+  GridRowParams,
+  GridToolbar,
+} from '@mui/x-data-grid'
+import { getRoot } from 'mobx-state-tree'
+import React, { useCallback, useMemo, useState } from 'react'
+
+import { useUsers } from './'
+
+// const users = [
+//   { id: 'user0123', email: '0123@demo.com', role: 'read-only' },
+//   { id: 'user4567', email: '4567@demo.com', role: 'user' },
+//   { id: 'user8910', email: '8910@demo.com', role: 'admin' },
+// ]
+
+interface ManageUsersProps {
+  session: AbstractSessionModel
+  handleClose(): void
+}
+
+export function ManageUsers({ session, handleClose }: ManageUsersProps) {
+  const { internetAccounts } = getRoot(session) as AppRootModel
+  const [errorMessage, setErrorMessage] = useState('')
+
+  const users = useUsers(internetAccounts, setErrorMessage)
+  type Row = typeof users[number]
+  const [rows, setRows] = useState<Row[]>(users)
```

Comment:
React's behavior here might not be intuitive, this will set `rows` to whatever `users` is the first time the code runs, but won't update it, so it will always be an empty array. You could probably use `users` directly without an intermediate `rows` state.